### ISSUE TITLE
Allow disabling depth writes when rendering

### ISSUE THREAD

**Issue by JohnDoneth** [phaazon/luminance-rs#400]
Sometimes it's useful to only write to the color buffer and not the depth buffer for things like skyboxes which is where I ran into this limitation.

Calling ...
```rust
unsafe {
    gl::DepthMask(false as u8);
};
```
... before rendering the skybox and then ...
```rust
unsafe {
    gl::DepthMask(true as u8);
};
```
... produces the correct result.

**Comment by phaazon** [comment]
That should be easy to add! Thanks! However, for your skybox problem: can’t you simply render all your scene, and then render your skybox?

**Comment by phaazon** [comment]
This is available on [luminance-0.40](https://crates.io/crates/luminance/0.40.0). Thanks for the feature request. :slightly_smiling_face:
